### DISCUSSION TITLE
Don't disable buttons on Github because there's no way to give feeback for why the button is disabled

### DISCUSSION THREAD

**Discussion by ranka23** [community/community#12820] [Other Feature Feedback, Questions, & Ideas]
If buttons are disabled there's no way to give the user feedback or display an error message for why the button is disabled. Even when there's a lack of permissions 

eg: The Submit new Issue button is disabled until a title is provided for the issue. This can cause confusion in users. Instead of disabling the button. Keep it active, and when the user clicks it, show a message that says Title is required to submit an issue, and don't make the api call.

I think it's just poor UI/UX experience to disable buttons, inputs, select etc.

![Screenshot from 2022-03-12 11-41-02](https://user-images.githubusercontent.com/35439798/158226923-b8bc58d5-ac7b-441e-abd9-5cbc7a029879.png)

**Comment by z3ntu** [comment]
Just experienced this filling out a long Issue Form (https://github.com/nextcloud/android/blob/master/.github/ISSUE_TEMPLATE/bug_report.yml) and the "Submit new issue" was disabled and I couldn't figure out why. Only after manually removing the html attributes making it disabled and clicking it, I noticed that I hadn't filled out the Title.

**Reply by xerces8** [reply]
Thanks for the hint.

I had this problem today, basically what this comment says: https://github.com/ckeditor/github-writer/issues/172#issuecomment-2065885583

Very poor UI design.

Is this here an bug/RFE or is there another place for that?
